cluster/addons/ip-masq-agent/ip-masq-agent.yaml

```diff
@@ -33,6 +33,8 @@ spec:
         volumeMounts:
           - name: config
             mountPath: /etc/config
+      nodeSelector:
+        beta.kubernetes.io/masq-agent-ds-ready: "true"            
```

Comment:
yeah, I asked https://github.com/kubernetes/kubernetes/pull/47764#issuecomment-309614020 which assume ip-masq-agent won't run since there is no node marking with the newly introduced label. But on my side, I was confused that NON_MASQUERADE_CIDR=0.0.0.0/0 is another label on Kubelet which tells Kubelet to stop masq-ing. :-( @mikedanese Thanks for spot the issue.